packages/features/bookings/Booker/components/AvailableTimeSlots.tsx

```diff
@@ -1,26 +1,23 @@
 import { useRef } from "react";
```

Comment:
Was thinking about naming - shouldn't we have "available-time-slots.tsx" instead of "AvailableTimeSlots" ? And same for folder structure "BookEventForm" -> "book-event-form". When searching in vs code we still can find them - like "book-event-form" folder is in this case so they are the same case.